pandas/tests/io/test_parquet.py

```diff
@@ -368,6 +368,12 @@ def test_multiindex_with_columns(self, pa_ge_070):
             check_round_trip(df, engine, read_kwargs={'columns': ['A', 'B']},
                              expected=df[['A', 'B']])
 
+    def test_write_ignoring_index(self):
+        """Ensure index=False omits the index from the written Parquet file."""
```

Comment:
* This should be a comment, not a docstring.
* Reference the issue number in the comment.

Reply:
Done!